consumer_cmake_find/conanfile.txt

```diff
@@ -1,10 +1,10 @@
 [requires]
-boost/1.67.0@conan/stable
-Poco/1.9.0@pocoproject/stable
+boost/1.72.0
+poco/1.9.4
 
 [generators]
-cmake_find_pakcage
```

Comment:
This bug was on purpose. To keep students alert, learn to interpret Conan error messages, etc.